src/core/api/__tests__/public_api.test.ts

```diff
@@ -0,0 +1,571 @@
+        expect(player.getVolume()).to.equal(0.8);
+        expect(videoElement.volume).to.equal(0.8);
+      });
+
+      it("should return false in isMute by default", () => {
+        expect(player.isMute()).to.equal(false);
+      });
+
+      it("should return true in isMute if the volume is equal to 0", () => {
+        const oldVolume = player.getVolume();
+
+        expect(player.isMute()).to.equal(false);
+
+        player.setVolume(0);
+        expect(player.isMute()).to.equal(true);
+        player.setVolume(oldVolume);
+        expect(player.isMute()).to.equal(false);
+
+        player.mute();
+        expect(player.isMute()).to.equal(true);
+        player.unMute();
+        expect(player.isMute()).to.equal(false);
+
+        player.mute();
+        expect(player.isMute()).to.equal(true);
+        player.setVolume(oldVolume);
+        expect(player.isMute()).to.equal(false);
+        player.unMute();
+        expect(player.isMute()).to.equal(false);
+
+        player.setVolume(oldVolume);
+      });
+    });
+
+    describe("setAudioBitrate/getManualAudioBitrate", () => {
+      it("should have a -1 manual audio bitrate by default", () => {
+        expect(player.getManualAudioBitrate()).to.equal(-1);
+      });
+
+      it("should update manual audio bitrate when calling setAudioBitrate", () => {
+        const oldManual = player.getManualAudioBitrate();
+
+        player.setAudioBitrate(84);
+        expect(player.getManualAudioBitrate()).to.equal(84);
+        player.setAudioBitrate(-1);
+        expect(player.getManualAudioBitrate()).to.equal(-1);
+        player.setAudioBitrate(0);
+        expect(player.getManualAudioBitrate()).to.equal(0);
+
+        player.setAudioBitrate(oldManual);
+        expect(player.getManualAudioBitrate()).to.equal(oldManual);
+      });
+    });
+
+    describe("setVideoBitrate/getManualVideoBitrate", () => {
+      it("should have a -1 manual video bitrate by default", () => {
+        expect(player.getManualVideoBitrate()).to.equal(-1);
+      });
+
+      it("should update manual video bitrate when calling setVideoBitrate", () => {
+        const oldManual = player.getManualVideoBitrate();
+
+        player.setVideoBitrate(84);
+        expect(player.getManualVideoBitrate()).to.equal(84);
+
+        player.setVideoBitrate(-1);
+        expect(player.getManualVideoBitrate()).to.equal(-1);
+
+        player.setVideoBitrate(0);
+        expect(player.getManualVideoBitrate()).to.equal(0);
+
+        player.setVideoBitrate(oldManual);
+        expect(player.getManualVideoBitrate()).to.equal(oldManual);
+      });
+    });
+
+    describe("setMaxVideoBitrate/getMaxVideoBitrate", () => {
+      /* tslint:disable:max-line-length */
+      it("should update the maximum video bitrate when calling setMaxVideoBitrate by default", () => {
+      /* tslint:enable:max-line-length */
+        const oldMax = player.getManualVideoBitrate();
+
+        expect(player.setMaxVideoBitrate(Infinity)).to.equal(undefined);
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Remove this use of the output from "setMaxVideoBitrate"; "setMaxVideoBitrate" doesn't return anything. ([typescript:S3699](https://sonarcloud.io/organizations/rx-player/rules#rule_key=typescript%3AS3699))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=414&issues=AWgOMLyH_MPaBDn0sxTG&open=AWgOMLyH_MPaBDn0sxTG)

[ISSUE_KEY]: AWgOMLyH_MPaBDn0sxTG